ygnmi/bench_test.go

```diff
@@ -28,24 +29,31 @@ import (
 	gpb "github.com/openconfig/gnmi/proto/gnmi"
 )
 
+// subClient will send count notifications, then return EOF
+// if count > len(resp), the last notification is sent repeatedly
 type subClient struct {
 	gpb.GNMI_SubscribeClient
-	resp *gpb.SubscribeResponse
-	sent bool
+	resp  []*gpb.SubscribeResponse
+	count int
```

Comment:
It's not immediately obvious to me what this means. I'm guessing something like "the number of non-EOF responses to be received by the client"?

Reply:
I meant the "count" field.

Reply:
renamed count to numResponses and added comment